docs/data/material/components/cards/cards.md

```diff
@@ -20,7 +20,7 @@ The Material UI Card component includes several complementary utility component
 - Card: a surface-level container for grouping related components.
 - Card Content: the wrapper for the Card content.
 - Card Header: an optional wrapper for the Card header.
-- Card Media: an optional container for displaying background images and gradient layers behind the Card Content.
+- Card Media: an optional container for displaying background images and gradient layers above the Card Content.
```

Comment:
This would be more accurate

```suggestion
- Card Media: an optional container for displaying images, videos, etc.

Reply:
Thank you for your suggestion. I agree that the description for Card Media could be more concise.